lib/ingress.js

```diff
@@ -80,9 +82,10 @@ function addIngressRuleIfNecessary(ruleName, functions, options) {
           annotations: {
```

Comment:
to avoid the linter issue you can use lodash:
```
annotations: _.merge(
  {
    'kubernetes.io/ingress.class': opts.ingress.class,
    [`${opts.ingress.class}.ingress.kubernetes.io/rewrite-target`]: '/',
  },
  opts.ingress.additionalAnnotations,
});
```